src/collects/seashell/backend/asan-error-parse.rkt

```diff
@@ -0,0 +1,344 @@
+#lang typed/racket
+
+(require/typed json [jsexpr->string (JSExpr -> String)]
+               [string->jsexpr (String -> JSExpr)]
+               [jsexpr? (Any -> Boolean)])
+(require/typed racket/string [string-prefix? (String String -> Boolean)])
```

Comment:
Can you use (rename-in ...) so it's clear you're not using the standard types for these functions?